welcome/index.adoc

```diff
@@ -6,10 +6,18 @@ include::modules/common-attributes.adoc[]
 [.lead]
 Welcome to the official {product-title} {product-version} documentation, where you can find information to help you learn about {product-title} and start exploring its features.
 
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated,openshift-online,openshift-aro[]
 To navigate the {product-title} {product-version} documentation, you can either
 
 * Use the left navigation bar to browse the documentation or
 * Select the activity that interests you from the contents of this Welcome page
+endif::[]
+
+ifdef::openshift-rosa[]
+To navigate the {product-title} (ROSA) documentation, use the left navigation bar.
+
+For documentation that is not ROSA-specific, see the link:https://docs.openshift.com/container-platform/4.7/welcome/index.html[OpenShift Container Platform documentation].
```

Comment:
Do you want this to link specifically to a single version of OCP (4.7) that has to be manually updated?